<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="launchAssemblyButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch Assembly</string>
    </property>
   </widget>
   <widget class="QPushButton" name="launchCppButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch C++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="cppBrowser">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>271</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="asmBrowser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>271</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="vertexNameLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="asmLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Assembly Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cppLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>5</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C++ Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>No file loaded</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="vertLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Starting Vertex Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="thrLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string># of Threads (C++ only):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="threadsNumLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
